code/backend/package.json

```diff
@@ -15,6 +15,10 @@
   },
   "devDependencies": {},
   "scripts": {
+    "testl": "DEVELOPMENT=true node template-generation/test/test-template-generation.js",
```

Comment:
- `test` not `testl`
- Also the environment variable `DEVELOPMENT` was removed from the test code, it's no longer necessary. I'll update any docs if they still have it.